node/network/approval-distribution/src/tests.rs

```diff
@@ -2283,7 +2285,8 @@ fn batch_test_round(message_count: usize) {
 	let mut state = State::default();
 
 	let (mut context, mut virtual_overseer) = test_helpers::make_subsystem_context(pool.clone());
-	let subsystem = ApprovalDistribution::new(Default::default());
+	let keysytore = test_helpers::mock::make_ferdie_keystore();
+	let subsystem = ApprovalDistribution::new(Default::default(), keysytore);
```

Comment:
Of course this is WIP. Just noticed strange var name "keysytore". Is it not by chance?